packages/c-live-region/src/live-region.ts

```diff
@@ -0,0 +1,148 @@
+function isDom() {
+  return Boolean(globalThis?.document)
+}
+const isBrowser = isDom()
+export interface LiveRegionOptions {
+  /**
+   * A unique id for the created live region element
+   */
+  id?: string
+  /**
+   * Used to mark a part of the page as "live" so that updates will
+   * be communicated to users by screen readers.
+   *
+   * - If set to `polite`: tells assistive technology to alert the user
+   * to this change when it has finished whatever it is currently doing
+   *
+   * - If set to `assertive`: tells assistive technology to interrupt whatever
+   * it is doing and alert the user to this change immediately
+   *
+   * @default "polite".
+   */
+  "aria-live"?: "polite" | "assertive"
+  /**
+   * The desired value of the role attribute
+   * @default "status"
+   */
+  role?: "status" | "alert" | "log"
+  /**
+   * Indicates what types of changes should be presented to the user.
+   * @default "all"
+   */
+  "aria-relevant"?:
```

Comment:
In the React package, there is the `React.AriaAttributes` type signature that is used to retrieve the types for the `aria-relevant` and `aria-atomic` attributes, instead of hard coding them here. 

Is their a Vue version of this signature??

Reply:
Ah I see. I don't think there is such a type in Vue.js. all of them usually fall under the `HTMLAttributes` type exported from Vue.

Reply:
Whoops! Forgot about that. 😅